contracts/DNSSECImpl.sol

```diff
@@ -268,38 +268,41 @@ contract DNSSECImpl is DNSSEC, Owned {
      * @dev Returns data about the RRs (if any) known to this oracle with the provided type and name.
      * @param dnstype The DNS record type to query.
      * @param name The name to query, in DNS label-sequence format.
-     * @return inception The unix timestamp at which the signature for this RRSET was created.
-     * @return inserted The unix timestamp at which this RRSET was inserted into the oracle.
      * @return hash The hash of the RRset that was inserted.
+     * @return inception The unix timestamp at which the signature for this RRSET was created.
      */
-    function rrdata(uint16 dnstype, bytes calldata name) external view returns (uint32, uint64, bytes20) {
+    function rrdata(uint16 dnstype, bytes calldata name) external view returns (bytes20, uint32, uint32) {
         RRSet storage result = rrsets[keccak256(name)][dnstype];
-        return (result.inception, result.inserted, result.hash);
```

Comment:
Could you preserve the original order here, please?

Also ,changing the signature here breaks other stuff - would you mind bumping the version number in package.json, and submitting a PR to ensdomains/dnsregistrar to update it to the new interface?

Reply:
> Also ,changing the signature here breaks other stuff - would you mind bumping the version number in package.json, and submitting a PR to ensdomains/dnsregistrar to update it to the new interface?

You bet: ensdomains/dnsregistrar#27 and ensdomains/dnsprovejs#20 are the corresponding pull requests. I'll bump the version number in package.json.

> Could you preserve the original order here, please?

Since we're changing the interface (`inserted` -> `expiration`) I'd like to make the new interface nicer for destructuring assignment: Clients (dnsregistrar, dnsprovejs) commonly need to check the hash and expiration, however you only need the inception if you're fast-failing an update, and only when performing an update, not a lookup.